src/client/dfs/dfs.c

```diff
@@ -1336,10 +1336,7 @@ dfs_cont_create_int(daos_handle_t poh, uuid_t *cuuid, bool uuid_is_set, uuid_t i
 	prop->dpp_entries[prop->dpp_nr - 1].dpe_type = DAOS_PROP_CO_LAYOUT_TYPE;
 	prop->dpp_entries[prop->dpp_nr - 1].dpe_val = DAOS_PROP_CO_LAYOUT_POSIX;
 
-	if (uuid_is_set)
```

Comment:
should we remove arg uuid_is_set? it looks i will be not used anywhere.

Reply:
I talked to Johann. Yes. We clean up compatibility code for dfs_cont_create() too. Both "bool uuid_is_set" and "uuid_t in_uuid" are removed now. Thank you!